WordPress/src/main/res/layout/my_site_fragment.xml

```diff
@@ -50,104 +50,78 @@
-                        tools:text="Example" />
-
-                    <org.wordpress.android.widgets.WPTextView
-                        android:id="@+id/my_site_subtitle_label"
+                        android:layout_alignTop="@+id/blavatar_container"
+                        android:layout_alignBottom="@+id/blavatar_container"
+                        android:layout_marginStart="@dimen/margin_extra_large"
+                        android:layout_toStartOf="@+id/switch_site"
+                        android:layout_toEndOf="@+id/blavatar_container" />
+
+                    <ImageButton
+                        android:id="@+id/switch_site"
                         android:layout_width="wrap_content"
                         android:layout_height="wrap_content"
-                        android:layout_gravity="center"
-                        android:layout_marginTop="@dimen/margin_extra_small"
-                        android:background="?attr/selectableItemBackground"
-                        android:ellipsize="end"
-                        android:gravity="center_vertical"
-                        android:maxLines="1"
-                        android:paddingStart="@dimen/margin_extra_large"
-                        android:paddingTop="@dimen/margin_extra_small"
-                        android:paddingEnd="@dimen/margin_extra_large"
-                        android:paddingBottom="@dimen/margin_extra_small"
-                        android:textAppearance="?attr/textAppearanceBody2"
-                        android:textColor="?attr/wpColorOnSurfaceMedium"
-                        tools:text="example.wordpress.com" />
-                </LinearLayout>
-
-                <com.google.android.material.button.MaterialButton
-                    android:id="@+id/switch_site"
-                    style="@style/Widget.MaterialComponents.Button.TextButton"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_gravity="center"
-                    android:layout_marginTop="@dimen/margin_extra_small"
-                    android:layout_marginBottom="@dimen/margin_extra_medium_large"
-                    android:ellipsize="end"
-                    android:gravity="center_vertical"
-                    android:maxLines="1"
-                    android:text="@string/my_site_btn_switch_site" />
+                        android:layout_alignParentEnd="true"
+                        android:layout_centerVertical="true"
+                        android:layout_marginStart="@dimen/margin_extra_large"
+                        android:background="?attr/selectableItemBackgroundBorderless"
+                        android:contentDescription="@string/my_site_btn_switch_site"
+                        android:src="@drawable/ic_chevron_down_white_24dp"
+                        app:tint="?attr/wpColorOnSurfaceMedium" />
```

Comment:
I didn't notice this the first time, but I believe we should give this button a larger touch area.

Giving it a padding of `12dp` and a margin of `4dp` should be enough. You'll probably need to remove the inner padding from the parent view and adjust the margin from the siblings, but I think it's worth it. Then you might also replace the background with a drawable [like this](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/res/drawable/ripple_oval.xml), to make the ripple bounded to its new size. 

Of course, consider all of that a merely suggestion (totally up to you!).

Reply:
Thank you, that's a great suggestion!